k-distribution/include/ktest-keq.mak

```diff
@@ -2,6 +2,7 @@
 MAKEFILE_PATH := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
 # path to the kompile binary of this distribuition
 KOMPILE=$(abspath $(MAKEFILE_PATH)/../bin/kompile)
+KOMPILE_BACKEND=java
```

Comment:
Don't you need a ? here to allow changing the backend?

Reply:
The keq testset always uses the Java backend for now. It's a different execution mode.